client/src/components/navigation/Navbar.js

```diff
@@ -34,7 +34,7 @@ const styles = theme => ({
     }
   },
   menuButton: {
-    [theme.breakpoints.up('sm')]: {
+    [theme.breakpoints.down('sm')]: {
       display: 'none'
```

Comment:
I think this should stay at up if I'm not mistaken, the hamburger menu is only for mobile versions

Reply:
My mistake - I was playing around with this to see how the breakpoints are working. I didn't mean to change this